src/summary/history/history-section.component.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { match } from "react-router";
+import SummarySectionHeader from "../summary-section-header.component";
+import SummarySectionCards from "../cards/summary-section-cards.component";
+import AllergyCard from "./allergy-card.component";
+import { getCurrentPatient } from "@openmrs/esm-api";
+
+export default function HistorySection(props: HistorySectionProps) {
+  const [currentPatient, setCurrentPatient] = React.useState(null);
+
+  React.useEffect(() => {
+    const subscription = getCurrentPatient().subscribe(patient =>
+      setCurrentPatient(patient)
+    );
+
+    return () => subscription.unsubscribe();
+  });
+
+  return (
+    <>
+      <SummarySectionHeader match={props.match} name="History" />
+      <SummarySectionCards match={props.match}>
+        {currentPatient != null && (
```

Comment:
It is best practice to always do `===` or `!==` instead of `==` or `!=`. See [this explanation](https://www.impressivewebs.com/why-use-triple-equals-javascipt/).

In your case, though, a truthiness check would be even better and easier:

```suggestion
        {currentPatient && (
```